templates/layout.html

```diff
@@ -42,33 +42,11 @@
 
   </head>
   <body>
-    <nav class="navbar navbar-dark navbar-fixed-top bg-inverse">
-      <div class="container">
-        <button class="navbar-toggler hidden-md-up" type="button" data-toggle="collapse" data-target="#collapsingNavbar">
-        &#9776;
-        </button>
-        <div class="collapse navbar-toggleable-sm" id="collapsingNavbar">
-          <a class="navbar-brand" href="/">BeeWare</a>
-          <ul class="nav navbar-nav">
-            <li class="nav-item{% if this.is_child_of('/project') %} active{% endif %}">
-              <a class="nav-link" href="/project/">Project{% if this.is_child_of('/project') %}<span class="sr-only">(current)</span>{% endif %}</a>
-            </li>
-            <li class="nav-item{% if this.is_child_of('/news') %} active{% endif %}">
-              <a class="nav-link" href="/news/">News{% if this.is_child_of('/news') %}<span class="sr-only">(current)</span>{% endif %}</a>
-            </li>
-            <li class="nav-item{% if this.is_child_of('/community') %} active{% endif %}">
-              <a class="nav-link" href="/community/">Community{% if this.is_child_of('/community') %}<span class="sr-only">(current)</span>{% endif %}</a>
-            </li>
-            <li class="nav-item{% if this.is_child_of('/contributing') %} active{% endif %}">
-              <a class="nav-link" href="/contributing/">Contributing{% if this.is_child_of('/contributing') %}<span class="sr-only">(current)</span>{% endif %}</a>
-            </li>
-            <li class="nav-item{% if this.is_child_of('/contact') %} active{% endif %}">
-              <a class="nav-link" href="/contact/">Contact{% if this.is_child_of('/contact') %}<span class="sr-only">(current)</span>{% endif %}</a>
-            </li>
-          </ul>
-        </div>
-      </div>
-    </nav>
+    {% if this.alt == 'pt' %}
```

Comment:
Is there any way to make this automated, rather than requiring a manual rollout for every translation language?